assembl/static2/js/app/components/administration/landingPage/manageModules.jsx

```diff
@@ -102,9 +191,17 @@ const mapDispatchToProps = dispatch => ({
   moveModuleDown: id => dispatch(moveLandingPageModuleDown(id)),
   moveModuleUp: id => dispatch(moveLandingPageModuleUp(id)),
   toggleModule: id => dispatch(toggleLandingPageModule(id)),
-  createModule: (nextOrder, identifier = 'INTRODUCTION') => {
+  createModule: (nextOrder, numberOfDuplicatesModules, identifier = MODULES_IDENTIFIERS.introduction) => {
     const newId = createRandomId();
-    return dispatch(createLandingPageModules(newId, identifier, nextOrder));
+    return dispatch(
+      createLandingPageModules(
```

Comment:
why is this called `createLandingPageModules`? shouldn't it be `createLandingPageModule`?

Reply:
True